### DISCUSSION TITLE
config tabufline icon

### DISCUSSION THREAD

**Discussion by Song-Tianxiang** [NvChad/NvChad#2552] [Q&A] [answered]
Hello, I want to know if there exist a way to config tabufline icon.
<img width="250" alt="Screenshot 2023-12-07 at 10 50 13 PM" src="https://github.com/NvChad/NvChad/assets/149415622/690a4bab-0f77-4467-92d0-99a99b5a5b6c">

I found tabufline modules config from nvchad website. But I'm afraid of maintain the whole tabufline modules file just for an icon.

**Comment by siduck** [comment, marked as answer]
@Song-Tianxiang not possible without overrides at this moment, probably in v3.0
